packages/config/next/next.config.js

```diff
@@ -0,0 +1,42 @@
+/** @type {import('next').NextConfig} */
+const { withSentryConfig } = require("@sentry/nextjs");
+
+const nextConfig = {};
+
+module.exports = nextConfig;
+
+module.exports = withSentryConfig(
+  module.exports,
```

Comment:
This should be good enough for now
```suggestion
module.exports = withSentryConfig(
  {},
```